esp-hal/src/dma/pdma.rs

```diff
@@ -12,7 +12,7 @@
 //! [I2S]: ../i2s/index.html
 
 use critical_section::CriticalSection;
```

Comment:
Should this file follow the convention in the hal and be moved to `pdma/mod.rs` as  well?